test_runner.py

```diff
@@ -244,7 +240,7 @@ def run_test(test_flavor: OverrideDefinitions, full_path: str):
 
 
 def run_tests(args):
-    integration_tests_flavors = build_test_list(args)
+    integration_tests_flavors = build_test_list()
```

Comment:
why we remove the `args` here, is it not getting used?

Reply:
that's right.